cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags)
+	logger.Infof("starting rook %s with arguments '%s'", version.Version, strings.Join(os.Args, " "))
+	logger.Infof("flag values: %s", strings.Join(flagValues, ", "))
```

Comment:
I have added functionality to filter all flags with "secret" in their name and added a unit test for it.